pkgs/development/python-modules/rss2email-sendmail-default.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/rss2email/config.py b/rss2email/config.py
+index 5c758bd..d73c8c2 100644
+--- a/rss2email/config.py
++++ b/rss2email/config.py
+@@ -186,7 +186,7 @@ CONFIG['DEFAULT'] = _collections.OrderedDict((
+         ('email-protocol', 'sendmail'),
+         # True: Use SMTP_SERVER to send mail.
+         # Sendmail (or compatible) configuration
+-        ('sendmail', '/usr/sbin/sendmail'),  # Path to sendmail (or compatible)
++        ('sendmail', '/run/current-system/sw/bin/sendmail'),  # Path to sendmail (or compatible)
```

Comment:
This won’t work with non-nixos. It should be patched to either `/usr/bin/env sendmail` or to a package directly.